parcels/interaction/brute_force.py

```diff
@@ -0,0 +1,18 @@
+from parcels.interaction.base_neighbor import BaseFlatNeighborSearch
```

Comment:
I don't particularly mind, but don't all of the files then become rather long? It is also possible to use a subpackage `interaction.neighborsearch`, so that it becomes a bit more clearer (at the expense of breaking the rules I suppose)?

Reply:
Done